<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XML2Print</class>
 <widget class="QDialog" name="XML2Print">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Формирование печатной формы документа</string>
  </property>
  <property name="windowIcon">
   <iconset resource="..\resources.qrc">
    <normaloff>:/plugins/openland/icons/print.png</normaloff>:/plugins/openland/icons/print.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Вид документа&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditFileXML">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonFileXML">
     <property name="text">
      <string>Выбрать...</string>
     </property>
     <property name="icon">
      <iconset resource="..\resources.qrc">
       <normaloff>:/plugins/openland/icons/find_search.png</normaloff>:/plugins/openland/icons/find_search.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Выполнение&lt;br/&gt;операций &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Файл XML&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonPrint">
     <property name="text">
      <string>Создать</string>
     </property>
     <property name="icon">
      <iconset resource="..\resources.qrc">
       <normaloff>:/plugins/openland/icons/print.png</normaloff>:/plugins/openland/icons/print.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBoxIsOpenDoc">
     <property name="text">
      <string>Открыть документ после создания</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="listWidgetEvents"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxDocumentType"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Закрыть</string>
     </property>
     <property name="icon">
      <iconset resource="..\resources.qrc">
       <normaloff>:/plugins/openland/icons/close.png</normaloff>:/plugins/openland/icons/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxDocumentType</tabstop>
  <tabstop>lineEditFileXML</tabstop>
  <tabstop>pushButtonFileXML</tabstop>
  <tabstop>listWidgetEvents</tabstop>
  <tabstop>pushButtonPrint</tabstop>
  <tabstop>checkBoxIsOpenDoc</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="..\resources.qrc"/>
 </resources>
 <connections/>
</ui>
